rosdep/python.yaml

```diff
@@ -689,6 +689,18 @@ python-evdev:
     vivid: [python-evdev]
     wily: [python-evdev]
     xenial: [python-evdev]
+python-expiringdict:
+  debian:
+    jessie: [python-expiringdict]
+    sid: [python-expiringdict]
```

Comment:
Ok, is [this place](http://www.ros.org/reps/rep-0003.html) the reference to find targeted versions or is there something else to quickly find which rosdep keys should be added ?
Thanks !